tests/foreman/ui/test_host.py

```diff
@@ -438,14 +438,14 @@ def test_positive_end_to_end(session, module_host_template, module_org, module_g
             module_host_template.operatingsystem.name, module_host_template.operatingsystem.major
         )
         assert os_name in displayed_host['Operating System']
-        assert displayed_host['Installed'] == '-'
+        assert displayed_host['Installed'] == 'N/A'
         # update
         session.host.update(host_name, {'host.name': new_name})
-        assert not session.host.search(host_name)
+        assert not entities.Host().search(query={'search': f'name="{host_name}"'})
```

Comment:
is this also failing ? I was thinking when you do not have hosts resources/entities then search is will not be displayed. In your case you have at-least 1 host

Reply:
Good question, I will look into this :)

Reply:
but definitely not a blocker, because this way it is faster and still reliable :)

Reply:
updated, you were correct :+1: